omero/sysadmins/unix/server-installation.txt

```diff
@@ -136,9 +136,8 @@ If possible, install one of the following packages:
 +-----------+---------------------------+
 
 OMERO works with the OpenJDK JRE provided by most systems, or with
-Oracle Java. Version 6 or later is required. Version 7 or later is
-recommended (version 6 is end of line and no longer has updates or
-security support). **OpenJDK version 8 is not currently supported**.
+Oracle Java. Version 7 or later is
+recommended. **OpenJDK version 8 is not currently supported**.
```

Comment:
Some versions e.g. in Ubuntu 14.10 have completely broken jpeg/image loading support.  When you run Insight all you see is a bunch of black squares for every image.  On FreeBSD it does work, with a different OpenJDK release, so it's not that we don't support it, it's that some versions in common use are unusable at present.